File: C/242_Valid_Anagram.c
bool isAnagram(char* s, char* t) {
    int salpha[256], talpha[256];
    for(int i = 0; i < 256; i++)
    {
        salpha[i] = 0;
        talpha[i] = 0;
    }
    for(int i = 0; i < strlen(s); i++)
        salpha[s[i]]++;
    for(int i = 0; i < strlen(t); i++)
        talpha[t[i]]++;
    for(int i = 0; i < 256; i++)
        if(salpha[i] != talpha[i])
            return false;
    return true;
}

File: Python/242_Valid_Anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        l1 = []
        l2 = []
        for c in s:
            l1.append(c)
        for c in t:
            l2.append(c)
        l1.sort()
        l2.sort()
        if l1 == l2:
            return True
        return False
